src/sentry/rules/processing/delayed_processing.py

```diff
@@ -244,7 +350,6 @@ def _passes_comparison(
         return False
 
     calculated_value = query_values[0]
-    # If there's a second query we must have a percent comparison condition.
```

Comment:
can this comment also be added back?

Reply:
👍 yeah - a can do.

That said, i don't see a lot of value in this component as it's saying the same thing as the code. (query length is 2, than it's a percent_increase comparison). i wonder if there's a better comment about why we're making this decision or why this is being handled in this part of the code (vs elsewhere)

Reply:
Hmm that's true, we could replace the comment and code with just:
```Python
if condition_data.get("comparisonType") == ComparisonType.PERCENT:
```

Reply:
LOVE that. updating.